.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

```suggestion
node_modules
.env
```
Don't forget to include .env so you protect your environment variables!